### ISSUE TITLE
Settings not saved

### ISSUE THREAD

**Issue by abhyaasa** [driftyco/ionic-lab-issues#53]
The settings view (gear icon) selections aren't saved from session to session (even when logged in).

**Comment by lucasyvas** [comment]
Indeed. It also doesn't remember to keep you logged in. I'm using Windows. It seems a little rushed.
